content/user/ui/settings/Integrations.md

```diff
@@ -80,11 +83,11 @@ You are connected!
 
 ## Bugtrackers
 
-TeamServer can connect to bugtrackers in order to streamline the remediation process. Individual or groups of discovered vulnerabilities can be sent directly to a configured bugtracker so that development groups may implement a fix.
+Contrast can connect to bugtrackers in order to streamline the remediation process. Individual or groups of discovered vulnerabilities can be sent directly to a configured bugtracker so that development groups may implement a fix.
```

Comment:
Should we link to our article about managing the lifecycle of a vulnerability (remediation)?